src/exporter.js

```diff
@@ -103,7 +103,7 @@ export const blockToHTML = (block) => {
     case Block.CODE:
       return {
         element: <pre className={`md-block-${blockType}`} />,
-        nest: <pre className="md-block-code-container" />,
+        nest: <div className="md-block-code-container" />,
```

Comment:
Making it div will also need you to add some css in `basic.css` so the exported code block looks like a code block in browser. Any reason for changing it to `div`?

Reply:
The first 'pre' is not necessary. Then the HTML code is cleaner like that.